src/pages/explainer page

```diff
@@ -0,0 +1,33 @@
+# How Open Source Can Help You Save Money and Time?
+
+
+## What is Open Source?
+
+Open source is products and tools containing source code and designs that the public can modify and change. Think of them as Google Slides or Microsoft PowerPoint presentations you create with your team for business meetings, but for software engineers, backend engineers, frontend developers, and other folks from different industries.
+
+
+## Why do Products Become Open Source?
+
+
+
+1. To interact with consumers: By making the designs and code open source, consumers have a more direct way to improve the products they enjoy, which in turn can save your company time and effort in creating troubleshooting guides. 
+2. To reach more demographics: Open source products tend to gain their following through social media, so this aspect of these tools can help increase brand awareness. 
+3. To build loyalty and trust: Since open source products allows people to see the source code. This form transparency help to build trust between consumers and the product’s creators.
+
+
+## Why do companies often choose to consume open source?
+
+
+
+1. Innovation: Since open source software often entails users and developers creating 
```

Comment:
Yeah, I was struggling on how to describe open source being innovative without repeating the points I made previously. Any suggestions @JoshuaKGoldberg?

Reply:
Maybe... because open source software allows anybody to propose changes, it allows participation from a wide variety of contributions. This means many more contributors can propose ideas and improve the project.

Reply:
That's a good one! :)